configuration.md

```diff
@@ -63,6 +65,7 @@ The CNI plugin may need to authenticate with the Kubernetes API server. The foll
 * `k8s_client_key`
 * `k8s_certificate_authority`
 	* Verifying the API certificate against a CA only works if connecting to the API server using a hostname.
+* `kubeconfig`
```

Comment:
I know the details on the other bits are pretty sparse, but they are reasonably self explanatory. It would be nice to at least put a bullet saying this should be a filename.